### PR TITLE
Iqra landing page

### PR DESCRIPTION
**Pull request by iqra-javed**
# Description

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
skeletonized all pages and added Access Control
Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

Tested access control on local host

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] My code has been reviewed by at least one peer
- [X] My changes generate no new warnings

### CODE REVIEW

File: client/src/actions/index.js [hunk 1]
```diff
@@ -49,3 +49,10 @@ export const login = data => {
       });
   };
 };
+
+export const logout = () => {
```
Comment (rusticlion) [inline client/src/actions/index.js:53]: Note that this action does not require input or deliver a payload - it simply wipes out the JWT from localStorage and makes a request to the reducer to clear out the auth state.

File: client/src/components/AccessControl.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+
+export default (ComposedComponent) => {
```
Comment (rusticlion) [inline client/src/components/AccessControl.js:5]: This higher-order function/component works as a "lock" on whatever component it is invoked on, checking the user's state for authorization at render-time and redirecting them if they are not authorized.
We will expand this component's functionality to also handle reauthorizing with the token on page reload.

File: client/src/index.js [hunk 3]
```diff
@@ -17,7 +26,16 @@ const store = createStoreWithMiddleware(combinedReducer);
 ReactDOM.render(
   <Provider store={store}>
     <Router>
-      <App />
+      <div>
```
Comment (rusticlion) [inline client/src/index.js:29]: The routes have been moved to the same level as the router, in `index.js`, obsoleting the App.js file. Each of the wireframe pages is now skeletonize and ready for further development. They will all be reachable from within the app with the buildout of a Navbar.

**Review by rusticlion — APPROVED** [review]
Added HoC for access control based on authorization and skeletonized all page routes. Removed create-react-app cruft.

**Review by emukupa — APPROVED** [review]
great work 👍

**Pull request merged**